homeassistant/components/ecobee/climate.py

```diff
@@ -287,6 +297,48 @@ def resume_program_set_service(service):
         "set_occupancy_modes",
     )
 
+    platform.async_register_entity_service(
+        SERVICE_SET_VENTILATOR_MODE,
+        {
+            vol.Required(ATTR_VENTILATOR_MODE): vol.Any(
+                "on", "off", "auto", "minontime"
+            ),
+        },
+        Thermostat.set_ventilator_mode.__name__,
```

Comment:
I'd recommend specifying the `"set_ventilator_mode"` string here, and, likewise, specifying the name strings in the other `async_register_entity_service` calls . Mainly, this keeps the usage consistent across all of these registration calls.

Reply:
Good point, will do.